Tests/TuistGeneratorAcceptanceTests/GenerateAcceptanceTests.swift

```diff
@@ -490,6 +490,26 @@ final class GenerateAcceptanceTestiOSAppWithExtensions: TuistAcceptanceTestCase
     }
 }
 
+final class GenerateAcceptanceTestiOSAppWithExtensionHavingUnitTestTarget: TuistAcceptanceTestCase {
+    func test_ios_app_with_extension_having_test_target() async throws {
+        try setUpFixture(.iosAppWithExtensionHavingUnitTest)
+        try await run(GenerateCommand.self)
+        try await run(BuildCommand.self, "App")
+
+        try await XCTAssertProductWithDestinationContainsExtension(
+            "App.app",
+            destination: "Debug-iphonesimulator",
+            extension: "AppExtension"
+        )
+        try await XCTAssertProductWithDestinationContainsExtension(
+            "App.app",
+            destination: "Debug-iphonesimulator",
+            extension: "AppExtension"
+        )
+        try await run(TestCommand.self, "--test-targets", "AppExtensionTests/ExtensionTests")
```

Comment:
Does this need to run on a simulator? If so, I'd skip it as booting up the simulator on the CI takes a long time.

Reply:
I tried `try await run(TestCommand.self, "AppExtensionTests/ExtensionTests")` and test fails

Reply:
When running locally it doesn't boot up simulator. I used macos as scheme for this test scheme. Could you tell me which flag would skip the simulator. Looking at the UT for extension I don't think this needs simulator to be booted. But not sure if AppExtension needs simulator or not

Reply:
> When running locally it doesn't boot up simulator. I used macos as scheme for this test scheme. Could you tell me which flag would skip the simulator. Looking at the UT for extension I don't think this needs simulator to be booted. But not sure if AppExtension needs simulator or not

That's fine then, was just checking 👍